tensorflow_datasets/core/dataset_utils.py

```diff
@@ -156,14 +171,15 @@ def _build_ds_from_instruction(instruction, ds_from_file_fn):
   return ds
 
 
-def _eager_dataset_iterator(dataset):
+def _eager_dataset_iterator(dataset: tf.data.Dataset):
   for item in dataset:
     flat = tf.nest.flatten(item)
     flat = [t if isinstance(t, tf.RaggedTensor) else t.numpy() for t in flat]
     yield tf.nest.pack_sequence_as(item, flat)
 
 
-def _graph_dataset_iterator(ds_iter, graph=None):
+def _graph_dataset_iterator(ds_iter: tf.data.Iterator,
```

Comment:
`tf.compat.v1.Iterator `. Did pytype run on this file ? Iterator don't exists anymore in TF 2.

Reply:
Sorry. My bad. Changed.